OpenRA.Mods.Common/Widgets/Logic/MusicPlayerLogic.cs

```diff
@@ -36,29 +36,30 @@ public MusicPlayerLogic(Widget widget, Ruleset modRules, World world, Action onE
 			BuildMusicTable();
 
 			Func<bool> noMusic = () => !musicPlaylist.IsMusicAvailable;
+			Func<bool> disableControls = () => !musicPlaylist.IsMusicAvailable || musicPlaylist.AmbientMusic;
```

Comment:
My debugger throws garble to this - although it compiles, from my tests, I'd assume the code only checks for `IsMusicAvailable`. Dun ask me why, I think I really can't grasp this `Func<bool>`.